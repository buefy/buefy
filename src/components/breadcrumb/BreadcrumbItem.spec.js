import { mount } from '@vue/test-utils'
import BBreadcrumbItem from '@components/breadcrumb/BreadcrumbItem'

describe('BBreadcrumbItem', () => {
    let wrapper
    beforeEach(() => {
        wrapper = mount(BBreadcrumbItem, {
            propsData: {
                tag: 'a'
            }
        })
    })

    it('is called', () => {
        expect(wrapper.isVueInstance()).toBeTruthy()
    })

    it('render correctly', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })

    it('should have a li tag', () => {
        expect(wrapper.contains('li')).toBeTruthy()
    })
})
