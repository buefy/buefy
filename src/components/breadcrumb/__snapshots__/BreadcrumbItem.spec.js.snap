// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BBreadcrumbItem render correctly 1`] = `<li class=""><a></a></li>`;
